readthedocs/projects/views/base.py

```diff
@@ -86,26 +83,17 @@ def get_form(self, data=None, files=None, **kwargs):
 
 class ProjectSpamMixin:
 
-    """Protects POST views from spammers."""
-
-    def post(self, request, *args, **kwargs):
-        log.bind(user_username=request.user.username)
-        if request.user.profile.banned:
-            log.info('Rejecting project POST from shadowbanned user.')
-            return HttpResponseRedirect(self.get_failure_url())
-        try:
-            return super().post(request, *args, **kwargs)
-        except ProjectSpamError:
-            date_maturity = timezone.now() - timedelta(
-                days=settings.USER_MATURITY_DAYS
-            )
-            if request.user.date_joined > date_maturity:
-                request.user.profile.banned = True
```

Comment:
So we will only have `banned` users manually now, correct? That seems fine.

Reply:
Yes. I didn't know about this code and I was surprised when I saw it yesterday. I think it wasn't working, tho. We can get back to "auto ban" in the future if we find a reliable way of doing it.